errorHandlingControllers.js

```diff
@@ -2,34 +2,43 @@ exports.handleInvalidPath = (req, res, next) => {
     res.status(404).send({ msg: 'Invalid path entered. Please check your URL and try again.'});
   };
 
-exports.handleCustomErrors = (err, req, res, next) => {
-    if (err.status && err.msg) {
-        res.status(err.status).send({ msg: err.msg});
+exports.handlePSQLs = (err, req, res, next) => {
+    if (err.code === '22P02' && !err.detail) {
+        res.status(400).send(res.statusMessage = "Bad article_id!");
```

Comment:
readability is improved by sending the object directly: `.send({statusMessage: "Bad article_id"})`